Lib/sysconfig/__init__.py

```diff
@@ -401,9 +401,20 @@ def _init_non_posix(vars):
     vars['BINLIBDEST'] = get_path('platstdlib')
     vars['INCLUDEPY'] = get_path('include')
 
-    # Add EXT_SUFFIX, SOABI, and Py_GIL_DISABLED
+    # Add EXT_SUFFIX, SOABI, Py_DEBUG, and Py_GIL_DISABLED
     vars.update(_sysconfig.config_vars())
 
+    # NOTE: ABIFLAGS is only an emulated value. It is not present during build
+    #       on Windows. sys.abiflags is absent on Windows and vars['abiflags']
+    #       is already widely used to calculate paths, so it should remain an
+    #       empty string.
+    vars['ABIFLAGS'] = ''.join(
+        (
+            't' if vars['Py_GIL_DISABLED'] else '',
+            '_d' if vars['Py_DEBUG'] else '',
+        ),
+    )
```

Comment:
This is stylistic, so only apply if you agree it looks nicer:

```suggestion
    vars['ABIFLAGS'] = ''.join((
        't' if vars['Py_GIL_DISABLED'] else '',
        '_d' if vars['Py_DEBUG'] else '',
    ))
```

Reply:
I'd keep this because it is a recommended style by `ruff format` or `black + flake8-commas`, which are common lint tools widely used in the Python community.